db/mysql/schema.sql

```diff
@@ -286,6 +300,16 @@ CREATE TABLE `sch_usr_notification_email`
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 CREATE INDEX `ix_SCH_USR_NOTIFICATION_EMAIL_email_id` ON `sch_usr_notification_email` (`email_id`);
 
+
+CREATE TABLE `organization`
```

Comment:
Table organization is created, but: 
1) it is not prepopulated with any value? If not, where is the value for the organization taken?
2) Is this table used somehow? There are couple of tables (e.g. ppl_model) including column organization_id, but it seems to me that it is always null. For example I created new pipeline and the organization_id for the new pipeline was null.